### ISSUE TITLE
Use of the term 'client' here is a confusing

### ISSUE THREAD

**Issue by markstos** [serverless/serverless-client-s3#17]
The description says 

_"A Serverless plugin that deploys a web client for your Serverless project to an S3 bucket,"_

This phrasing makes it sounds like this plugin puts an HTTP user agent ("web client") into an S3 bucket.  That doesn't seem to make sense.

My best understanding that this is actually a _client of an S3 server_  That is, it doesn't _put a client_ into S3, rather it _is_ a client deploys _static content_ into an S3 bucket. 

If that's right, the description and possibly some other phrasing could stand to updated. 

_"A Serverless plugin that publishes static content from your Serverless project to an S3 bucket,"_

**Comment by markstos** [comment]
Perhaps this is meaning of "web client" here:  Maybe you are referring to HTML pages which contain JavaScript that makes API calls to a web servers dynamic backend. Thus, the static HTML pages are the "client" of the backend REST APIs. 

I still think it's a confusing term here. As far as I can tell, this plugin doesn't care what kind of static assets you have, or whether they are part of a "client" that makes backend REST API calls.  It seems this plugin isn't really about "web clients", but simply about static content of any variety. 

For example in our case we have some static JSON we need to serve along side some dynamic JSON responses. The static JSON files aren't a "web client", but a tool like this might be useful to publish them.

I think use of "client" could be largely removed from this this README (and even the project name) and just replaced with "static" instead.

**Comment by vmadman** [comment]
@markstos Agreed.  I found this project thinking the same thing, that it would create and _then_ upload a custom API client to S3.  Your assessment is spot on, this is not a utility for API clients, but, rather, is simply a utility for deploying content to S3 during the Serverless deployment step.

Still, this project is useful, and while the project name (`serverless-client-s3`) is actually an appropriate name, I think the author should do a small amount to remove the confusion created by the README, unless the author would like to take this project down the road of web-client generation.

I say all of this with respect to the efforts of the author, and am certainly not looking to cause any grief.  The plugin is appreciated, either way.

**Comment by eahefnawy** [comment]
From a technical perspective, you're right. It just pushes any files to S3. However it's still meant to act as a client for your Serverless backend, and you can see this clearly in the README and specially that it's deploying the `dist` folder, which is generally associated with frontend websites.

Generally I like to include the `client` term cause it answers the question "what do I do with the front end of my web app" ... I'm imagining in the future there will be other type of clients as well, maybe client folder for mobile app clients, or IoT clients. Sky is the limit!
